src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -35,22 +35,23 @@ class HoldRequest extends React.Component {
    * submitRequest()
    * Client-side submit call.
    */
-  submitRequest(e, itemId) {
+  submitRequest(e, bibId, itemId) {
     e.preventDefault();
 
     axios
-      .get(`/api/newHold?id=${itemId}`)
+      .get(`/api/newHold?itemId=${itemId}`)
       .then(response => {
         if (response.data.error && response.data.error.status !== 200) {
-          this.context.router.push(`/hold/confirmation/${itemId}?errorMessage=` +
+          this.context.router.push(`/hold/confirmation/${bibId}-${itemId}?errorMessage=` +
```

Comment:
I'd argue between `bibId-itemId` or `bibId/itemId` but not in query form. An item will also always appear with a bib so it'd be more like `?bibId=b1234&itemId=i5678`.

We can update it later if we need to. React-router and Express router can handle both scenarios.